src/cmps/PokemonCard.tsx

```diff
@@ -24,14 +24,14 @@ const PokemonCardComponent = ({ pokemon, onClick }: PokemonCardProps) => {
           <PowerLevel>
             <Typography
               type="caption"
-              weight={700}
+              $weight={700}
```

Comment:
why do you need the $?

Reply:
Had a warning in the console: React does not recognize the `weight` prop on a DOM element.